Backpack/src/main/java/net/skyscanner/backpack/chip/BpkChip.kt

```diff
@@ -105,59 +92,85 @@ open class BpkChip @JvmOverloads constructor(
   }
 
   fun toggle() {
-    if (!disabled) {
+    if (isEnabled) {
       isSelected = !isSelected
     }
   }
 
   internal open fun provideAppearance(context: Context, attrs: AttributeSet?, defStyleAttr: Int): BpkChipAppearance =
-    BpkChipAppearances.Solid.fromAttrs(context, attrs, defStyleAttr)
+    BpkChipAppearances.fromAttrs(context, attrs, defStyleAttr)
 
   private fun updateStyle() {
-    this.background = appearance.background
-    setTextColor(appearance.text)
-    compoundDrawableTintList = appearance.text
+    elevation = if (style == Style.OnImage) resources.getDimension(R.dimen.bpkElevationSm) else 0f
```

Comment:
Silly non Android dev question is `0f` a thing or is this something else?

Reply:
Spoken to Maria in person, yes its a thing :) and not a mistake it means 'float' for anyone not familiar with it!